Terminal.Gui/View/Layout/DimAuto.cs

```diff
@@ -94,17 +95,17 @@ internal override int Calculate (int location, int superviewContentSize, View us
                 // [ ] DimView
                 if (dimension == Dimension.Width)
                 {
-                    subviews = us.Subviews.Where (v => v.X is not PosAnchorEnd
-                                                       && v.X is not PosAlign
-                                                       && v.X is not PosCenter
-                                                       && v.Width is not DimFill).ToList ();
+                    subviews = includedSubviews.Where (v => v.X is not PosAnchorEnd
+                                                           && v.X is not PosAlign
+                                                           // && v.X is not PosCenter
+                                                           && v.Width is not DimFill).ToList ();
                 }
                 else
                 {
-                    subviews = us.Subviews.Where (v => v.Y is not PosAnchorEnd
-                                                       && v.Y is not PosAlign
-                                                       && v.Y is not PosCenter
-                                                       && v.Height is not DimFill).ToList ();
+                    subviews = includedSubviews.Where (v => v.Y is not PosAnchorEnd
+                                                           && v.Y is not PosAlign
```

Comment:
Hm. Looks like this might be eligible for a pattern match. If it can't automatically make it for you, don't worry about it. But if you put the cursor on one of the `&&`s, it should be available in the lightbulb menu, if it can see how to do it.